### DISCUSSION TITLE
Can you add markers/goals to a bar chart with multiple bars

### DISCUSSION THREAD

**Discussion by jasmineibikunle1** [apexcharts/apexcharts.js#3690] [Q&A]
Hi everyone,

Wondering if I can get some help here. 

Is there a way to add goals/markers to bar charts with multiple data sets. 

![image](https://user-images.githubusercontent.com/36226730/223716797-df373472-4a92-4077-b9e7-38e1427357f8.png)

I'm trying to hack it but not sure which way to go about it.

**Comment by clyliston-axion** [comment]
I am following because I am curious too. I added a line but it is one value per group. Not very useful
